packages/app/src/__tests__/NexusPack.test.tsx

```diff
@@ -138,6 +160,42 @@ test('show error when `signal` entity is not found', async () => {
   expect(errorSpy).toHaveBeenCalledTimes(2); // React logs two stack traces
 });
 
+test('show error when `signal` entity is not a dataset', async () => {
+  await renderApp();
+
+  const errorSpy = mockConsoleMethod('error');
+  await selectExplorerNode('nexus_malformed/signal_not_dataset');
```

Comment:
I've added some more mock data that we can feature test, again as replacement for previous NeXus `utils` unit tests.